tests/js/server/aql/aql-optimizer-rule-late-document-materialization-arangosearch-primary-sort.inc

```diff
@@ -176,8 +176,8 @@
                     "LET e = SUM(FOR c IN " + vn + " LET p = CONCAT(c.foo, c.foo) RETURN p) " +
                     "SORT CONCAT(a, e) LIMIT 10 RETURN d";
         let plan = AQL_EXPLAIN(query).plan;
+        assertNotEqual(-1, plan.rules.indexOf(ruleName));
         if (!isCluster) {
```

Comment:
Should we then remove "isCluster" condition and indeed check late materialization rule usage in cluster in this case ?